From 3d559021b97e5b7189f4ee2d79eef9a661377e2c Mon Sep 17 00:00:00 2001
From: Adriana Reus <adriana.reus@intel.com>
Date: Fri, 30 Aug 2013 14:10:32 +0300
Subject: SQUISH Check calloc return value

On next rebase please squish into:
1dac056 Fastboot over TCP transport

Change-Id: I84f41a320733a8c7eccae4352479af8c6ee35e0a
Signed-off-by: Adriana Reus <adriana.reus@intel.com>
---
 fastboot/tcp.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/fastboot/tcp.c b/fastboot/tcp.c
index 9e1ad33..b4c490d 100644
--- a/fastboot/tcp.c
+++ b/fastboot/tcp.c
@@ -133,6 +133,10 @@ tcp_handle *tcp_open(const char *host)
     }
 
     tcp = calloc(1, sizeof(tcp_handle));
+    if (tcp == NULL) {
+        fprintf(stderr,"ERROR: Unable to allocate memory: %s\n",strerror(errno));
+        exit(1);
+    }
     tcp->sockfd = sockfd;
     return tcp;
 }
-- 
1.7.4.1

